app/budget/directives/budgetSummary/budgetSummary.html

```diff
@@ -217,8 +235,8 @@
 						<div class="budget-summary__cell-container"></div>
 					</td>
 					<td>
-						<div class="budget-summary__cell-container budget-summary__cell-container--last budget-summary__cell-container--last">
-							{{ toCurrency(selectedBudgetScenario.funds) }}
+						<div class="budget-summary__cell-container budget-summary__cell-container--last ">
+							{{ '-' + toCurrency(selectedBudgetScenario.funds) }}
```

Comment:
This may not make sense. They actually allow for negative fund amounts.

Reply:
Ah I see, that makes sense now as to why it was not implemented originally.